bundled/README.md

```diff
@@ -19,13 +19,22 @@ and licensed under the Boost Software License Version 1.0. See
 A full version of the library can be downloaded at http://www.boost.org/.
 
 
+cpp-taskflow-*
+--------------
+
+Contains the cpp-taskflow project licensed under the MIT license.
+
+A full version of the FunctionParser project can be downloaded at
+https://cpp-taskflow.github.io/#/
+
+
 muparser_*
 ----------
 
 Contains the MuParser project licensed under the MIT license.
 
 A full version of the FunctionParser project can be downloaded at
-http://muparser.beltoforion.de/.
+https://muparser.beltoforion.de/.
```

Comment:
"Your connection is not private"

Reply:
Wow. That worked two days ago. reverted.